src/app/js/admin/enrichment/EnrichmentLogs.js

```diff
@@ -0,0 +1,117 @@
+import React, { useContext, useEffect, useState } from 'react';
+import compose from 'recompose/compose';
+import translate from 'redux-polyglot/translate';
+import { polyglot as polyglotPropTypes } from '../../propTypes';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core';
+import { EnrichmentContext } from './enrichment-context';
+import { fromUser } from '../../sharedSelectors';
+import { connect } from 'react-redux';
+import theme from '../../theme';
+
+const useStyles = makeStyles({
+    LogsContainer: {
+        overflowY: 'auto',
+    },
+    LogsTitle: {
+        fontSize: '1.2rem',
+        fontWeight: 'bold',
+    },
+    Logs: {
+        width: '100%',
+        textAlign: 'left',
+    },
+    Log_info: {
+        color: theme.black.light,
+    },
+    Log_ok: {
+        color: theme.green.primary,
+        fontWeight: 'bold',
+    },
+    Log_error: {
+        color: theme.red.primary,
+        fontWeight: 'bold',
+    },
+});
+
+export const EnrichmentLogsComponent = ({ p: polyglot, optionsRequest }) => {
+    const { enrichment } = useContext(EnrichmentContext);
+
+    const url =
+        window.location.origin + `/api/job/log/enrichment/${enrichment?.jobId}`;
+    const options = optionsRequest({
+        url,
+        method: 'GET',
+    });
+
+    const [error, setError] = useState(null);
+    const [isLoaded, setIsLoaded] = useState(false);
+    const [logs, setLogs] = useState([]);
+
+    useEffect(() => {
+        enrichment?.jobId &&
+            fetch(url, options)
```

Comment:
I would extract this fetch in /js/lib as we will reuse this. Maybe a method like : `fetchData(url, method, payload)`